packages/colorpicker/lib/src/colorpicker.dart

```diff
@@ -0,0 +1,113 @@
+import 'package:colorpicker/src/constants/colors.dart';
+import 'package:colorpicker/src/components/color_comparison.dart';
+import 'package:colorpicker/src/components/slider.dart';
+import 'package:colorpicker/src/state/color_state.dart';
+import 'package:colorpicker/src/state/slider_position_state.dart';
+import 'package:colorpicker/utils/assets.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class ColorPicker extends ConsumerWidget {
+  const ColorPicker({
+    super.key,
+    required this.currentColor,
+    required this.onColorChanged,
```

Comment:
Actually we are building this as a separate package which will be used across our different applications, so won't this be a better approach to pass it as parameter and then use it as a widget, just like other packages on pub.dev do?
I'm not very sure about that but yeah, I thought it would make more sense this way.

Reply:
@bakicelebi opinion on this?

Reply:
Actually I had a discussion with @bakicelebi and since this package was to be built separately.. these changes were accepted